elements/pfe-content-set/src/pfe-content-set.js

```diff
@@ -316,15 +316,21 @@ class PfeContentSet extends PFElement {
    * @returns {boolean} True if the element provided is a header region
    */
   _isHeader(el) {
-    return !!(el.hasAttribute(`${this.tag}--header`) || el.tagName.match(/H[1-6]/));
+    if (typeof el.hasAttribute !== "undefined") {
```

Comment:
I need to put comments here.  We can add these to ensure that custom elements that are defined dynamically won't throw JS errors.